packages/suite/src/views/wallet/coinmarket/buy/components/BuyForm/Footer/index.tsx

```diff
@@ -120,7 +122,9 @@ const Footer = () => {
 
                                     return (
                                         <OptionLabel>
-                                            <Flag>{labelParts.flag}</Flag>
+                                            <FlagWrapper>
+                                                <Flag country={option.value as any} />
```

Comment:
Why have them as any if we have a list of flags?

Reply:
resolved